<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-1-g15f4949  Build Version: 1.6.7594.29634
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot name="LeRobot">

  <link name="world"/>
  <link name="Base">
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://lerobot/meshes/Base.STL" />
      </geometry>
      <material name="color">
        <color rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://lerobot/meshes/Base.STL" />
      </geometry>
    </collision>
  </link>

  <joint name="world_joint" type="fixed">
    <parent link="world"/>
    <child link="Base"/>
    <origin xyz="0.0 0.0 0.0" rpy="0 0 0"/>
  </joint>

  <link name="Shoulder_Rotation_Pitch">
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://lerobot/meshes/Shoulder_Rotation_Pitch.STL" />
      </geometry>
      <material name="color">
        <color rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://lerobot/meshes/Shoulder_Rotation_Pitch.STL" />
      </geometry>
    </collision>
  </link>
  <joint name="Shoulder_Rotation" type="revolute">
    <origin xyz="0 -0.0452 0.0181" rpy="1.5708 0 1.5708" />
    <parent link="Base" />
    <child link="Shoulder_Rotation_Pitch" />
    <axis xyz="0 1 0" />
    <limit lower="-1.57079632679" upper="1.57079632679" effort="10" velocity="10"/>
  </joint>
  <link name="Upper_Arm">
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://lerobot/meshes/Upper_Arm.STL" />
      </geometry>
      <material name="color">
        <color rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://lerobot/meshes/Upper_Arm.STL" />
      </geometry>
    </collision>
  </link>
  <joint name="Shoulder_Pitch" type="revolute">
    <origin xyz="0.000125 0.1086 0" rpy="3.1416 0 -1.5708" />
    <parent link="Shoulder_Rotation_Pitch" />
    <child link="Upper_Arm" />
    <axis xyz="0 0 1" />
    <limit lower="-1.57079632679" upper="1.57079632679" effort="10" velocity="10"/>
  </joint>
  <link name="Lower_Arm">
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://lerobot/meshes/Lower_Arm.STL" />
      </geometry>
      <material name="color">
        <color rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://lerobot/meshes/Lower_Arm.STL" />
      </geometry>
    </collision>
  </link>
  <joint name="Elbow" type="revolute">
    <origin xyz="-0.11238 0.0282 0" rpy="0 0 -1.5708" />
    <parent link="Upper_Arm" />
    <child link="Lower_Arm" />
    <axis xyz="0 0 1" />
    <limit lower="-1.57079632679" upper="1.57079632679" effort="10" velocity="10"/>
  </joint>
  <link name="Wrist_Pitch_Roll">
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://lerobot/meshes/Wrist_Pitch_Roll.STL" />
      </geometry>
      <material name="color">
        <color rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://lerobot/meshes/Wrist_Pitch_Roll.STL" />
      </geometry>
    </collision>
  </link>
  <joint name="Wrist_Pitch" type="revolute">
    <origin xyz="-0.1102 0.005375 0" rpy="0 1.5708 0" />
    <parent link="Lower_Arm" />
    <child link="Wrist_Pitch_Roll" />
    <axis xyz="1 0 0" />
    <limit lower="-1.57079632679" upper="1.57079632679" effort="10" velocity="10"/>
  </joint>
  <link name="Fixed_Gripper">
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://lerobot/meshes/Fixed_Gripper.STL" />
      </geometry>
      <material name="color">
        <color rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://lerobot/meshes/Fixed_Gripper.STL" />
      </geometry>
    </collision>
  </link>
  <joint name="Wrist_Roll" type="revolute">
    <origin xyz="0 0.002 -0.0545" rpy="3.1416 0 3.1416" />
    <parent link="Wrist_Pitch_Roll" />
    <child link="Fixed_Gripper" />
    <axis xyz="0 0 1" />
    <limit lower="-1.57079632679" upper="1.57079632679" effort="10" velocity="10"/>
  </joint>
  <link name="Moving_Jaw">
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://lerobot/meshes/Moving_Jaw.STL" />
      </geometry>
      <material name="color">
        <color rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://lerobot/meshes/Moving_Jaw.STL" />
      </geometry>
    </collision>
  </link>
  <joint name="Gripper" type="revolute">
    <origin xyz="0.0202 0 0.024375" rpy="-1.5708 0 0" />
    <parent link="Fixed_Gripper" />
    <child link="Moving_Jaw" />
    <axis xyz="0 0 1" />
    <limit lower="0" upper="1.57079632679" effort="10" velocity="10"/>
  </joint>
</robot>